install-worker.sh

```diff
@@ -20,22 +20,20 @@ sudo apt-get upgrade -y >/dev/null
 # Install necessary packages
 sudo apt-get install -y --no-install-recommends \
     awscli \
+    chrony \
     conntrack \
     curl \
     htop \
     jq \
     nfs-common \
     nmap \
-    ntp \
     socat \
     screen \
     sysstat \
     unzip \
     wget \
     >/dev/null
 
-sudo systemctl enable ntp
```

Comment:
Installing the `chrony` package automatically enables the `chronyd` service, so this line can be removed entirely.